benchmark.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/sh
+set -e
+
+if [ "$#" -ne 4 ] ; then
+    echo "Compare the speed of two different disasmtool versions"
+    echo "Usage $0 <first disasmtool> <second disasmtool> <input file> <iterations>"
+    exit 0
+fi
+
+FIRST=$1
+SECOND=$2
+INPUT=$3
+COUNT=$4
+
+if [ ! -x "$FIRST" ] ; then
+    echo "First program $FIRST does not exist or is not executable"
+    exit 1
+fi
+
+if [ ! -x "$SECOND" ] ; then
+    echo "Second program $SECOND does not exist or is not executable"
+    exit 1
+fi
+
+if [ ! -f "$INPUT" ] ; then
+    echo "Input file $INPUT does not exist"
+    exit 1
+fi
+
+case $COUNT in
+    ''|*[!0-9]*) echo "Iteration count $COUNT is not a number" ; exit 1 ;;
+    *) ;;
+esac
+
+truncate -s 0 before.txt
```

Comment:
Instead of using hard-coded output files, maybe use some names based on the disasmtool used - for example, `$FIRST.result` and `$SECOND.result`.

Reply:
Done. Also improved error messages and cleaned up white space.